secp256k1-sys/README.md

```diff
@@ -0,0 +1,34 @@
+secp256k1-sys
+=============
+
+
+This crate provides Rust definitions for the FFI structures and methods.
+
+
+## Vendoring
+
+The default build process is to build using the vendored libsecp256k1 sources in
+the depend folder. These sources are prefixed with a special
+rust-secp256k1-sys-specific prefix `rustsecp256k1_v1_2_3_`.
```

Comment:
Maybe point out which version number this is supposed to be. Of the `rust-secp256k1` crate or of the `-sys` crate? (I assume both... :))

Reply:
No, it's just the -sys crate. The rust-secp crate version can freely move around and make breaking changes in the API without a need to update -sys. -sys should only update if the upstream sources update.